src/test/fuzz/versionbits.cpp

```diff
@@ -305,10 +308,19 @@ FUZZ_TARGET_INIT(versionbits, initialize)
             assert(exp_state == ThresholdState::DEFINED);
         }
         break;
+    case ThresholdState::DELAYED:
     case ThresholdState::LOCKED_IN:
-        assert(exp_state == ThresholdState::STARTED);
-        assert(current_block->GetMedianTimePast() < checker.m_end);
-        assert(blocks_sig >= threshold);
+        if (state == ThresholdState::DELAYED) {
+            assert(current_block->GetMedianTimePast() < min_lock_in_time);
+        } else {
+            assert(min_lock_in_time <= current_block->GetMedianTimePast());
+        }
+        if (exp_state == ThresholdState::STARTED) {
+            assert(blocks_sig >= threshold);
+            assert(current_block->GetMedianTimePast() < checker.m_end);
+        } else {
+            assert(exp_state == ThresholdState::DELAYED);
+        }
```

Comment:
In commit "fuzz: test versionbits delayed lock in"

In case `case ThresholdState::FAILED` you want to check that `exp_state` isn't `ThresholdState::DELAYED` either (I think the current fuzz code won't prevent a transition DELAYED->FAILED).

Reply:
Ah, I updated that for LAST_CHANCE in https://github.com/ajtowns/bitcoin/commits/202103-bip9-uasf but didn't backfill to here.